src/components/Autocomplete/interfaces/AutocompleteHitInterface.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { ComponentInterface } from '@paper/core/interfaces/ComponentInterface';
+
+export interface AutocompleteHitInterface extends ComponentInterface {
+  active: boolean;
+  onClick: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  scrollTo: Function;
```

Comment:
```suggestion
  scrollTo: () => void;
```

You should use this instead of the Function type.